library/core/src/cell.rs

```diff
@@ -1428,33 +1448,41 @@ impl<'b> BorrowRef<'b> {
             // 1. It was = 0, i.e. it wasn't borrowed, and we are taking the first read borrow
             // 2. It was > 0 and < isize::MAX, i.e. there were read borrows, and isize
             //    is large enough to represent having one more read borrow
-            borrow.set(b);
+            borrow.replace(b);
             Some(BorrowRef { borrow })
         }
     }
+
+    /// `Clone` is not a `const_trait`, so work around that by making our own method
```

Comment:
```suggestion
    /// FIXME(const-hack): `Clone` is not a `const_trait`, so work around that by making our own method
```
This is a marker we use to make it easier to find and remove such hacks later